test_vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+java_versions:
+  - oracle-jdk7-installer
+  - openjdk-7-jdk
+java_default_version: java-8-oracle
```

Comment:
Shouldn't these variables go inside defaults/main.yml ? Also, we should pin the exact version.